doc/kconfig/conf.py

```diff
@@ -63,3 +63,4 @@ def setup(app):
     app.add_stylesheet("css/common.css")
     app.add_stylesheet("css/kconfig.css")
     app.add_js_file("js/removesearch.js")
+    app.add_js_file("js/ncs.js")
```

Comment:
At least for now, we should add the version switcher for the nrf repository only. Adding it to the other repos is confusing because the versions in the drop-down are not the versions for that respective repo, but for NCS.
Of course it would be great to be able to switch between different versions of each repo, but there is no one-to-one relation, so the same version of for example mcuboot might be used in several NCS releases. So let's just not have a version-switcher for those other repos.

Reply:
I am not sure I understood this correctly, so what I did is just to not display a versions list when the currently chosen set is not "nRF Connect SDK". I have not changed the versions shown in the docset chooser (at the bottom) nor the informative messages about browsing an old version. Let me know if you want those changed as well.

Reply:
That's what I meant, yes. 👍 
Sorry for being unclear.